src/react/components/common/tagsInput/tagsInput.test.tsx

```diff
@@ -50,29 +47,42 @@ describe("Tags Input Component", () => {
     it("create a new tag from text box - enter key", () => {
         const newTagName = "My new tag";
         wrapper.find("input").simulate("change", {target: {value: newTagName}});
-        wrapper.find("input").simulate("keyDown", {keyCode: 13}); // enter
+        wrapper.find("input").simulate("keyDown", {keyCode: KeyCodes.enter}); // enter
+        expect(onChangeHandler).toBeCalled();
+        expect(wrapper.state().tags).toHaveLength(originalTags.length + 1);
+        const newTagIndex = originalTags.length;
+        expect(wrapper.state().tags[newTagIndex].id).toEqual(newTagName);
+        expect(TagColors).toContain(wrapper.state().tags[newTagIndex].color);
+    });
+
+    it("create a new tag from text box - enter key", () => {
```

Comment:
Is this a repeat of the above test?

Reply:
Definitely was. Ctrl+V too many times :)